projects/packages/publicize/src/jetpack-social-settings/class-settings.php

```diff
@@ -56,31 +49,24 @@ class Settings {
 
 	/**
 	 * Migrate old options to the new settings. Previously SIG settings were stored in the
-	 * jetpack_social_image_generator_settings option. Now they are stored in the jetpack_social_settings
-	 * together with the auto conversion settings.
+	 * jetpack_social_image_generator_settings option. Now they are stored in the jetpack_social_settings.
 	 *
 	 * TODO: Work out if this is possible on plugin upgrade
 	 *
 	 * @return void
 	 */
 	private function migrate_old_option() {
```

Comment:
We could simply delete the call to migrate_old_option and remove this function. I don't think the migration to delete the option is worth doing given the risks  if some user decides to roll back a Jetpack version. I understand the motivation behind having this cleanup, but a few KBs lying around the site is better than multiple DB reads.

Reply:
If a user rolls back, then the plugin will fallback to the default settings which is fine I think.

Reply:
Yes, however, given the history  p58i-gbb-p2 I don't think it's worth the risk of doing this migration, certainly not in this PR where we are removing a bunch of other things as well.

Reply:
We have taken the precaution to not call delete_option all the time. So, I think it should be fine. If not, then we can always do a quick revert and address the concerns if any.

Reply:
Sounds good!